src/quemb/kbe/pbe.py

```diff
@@ -855,8 +853,8 @@ def initialize_pot(Nfrag, edge_idx):
     pot_ = []
 
     if not len(edge_idx) == 0:
-        for I in range(Nfrag):
```

Comment:
I am fine with this change, but note that we consciously deactivated [ambiguous-variable-name (E741)](https://docs.astral.sh/ruff/rules/ambiguous-variable-name/#ambiguous-variable-name-e741) [here](https://github.com/troyvvgroup/quemb/blob/main/.ruff.toml#L9)

It is just very common to have these letters in mathematical equations, hence in our code.